tapis-project/tapis-ui
# Task/TUI-17 - Clean up repo

## Overview: ##
**We'll need to update the github workflows to get this to pass the basic tests.** The build/install directory has been changed and is causing the tests to fail.

- Migrate react-tapis-helloworld branch to main branch in repo and github
- Hotloader is working
- Separate **tapis-redux**, **tapis-ui**, and **tapis-app** components.
- Organized **_common** ui components.
- Adjusted npm commands and aliases to work for new file structure


## Related Github Issues: ##

* [TUI-17](https://github.com/tapis-project/tapis-ui/issues/17)

## Summary of Changes: ##

## Testing Steps: ##
1. Get a clean pull of the repo and run `npm install` from the project root.
2. Check login capabilities
3. Check that Systems tab appears after login
4. npm run lint
5. npm run test
6. skim through configs for webpack, eslint, jest, package.json for anything that could be improved or rearranged.
